oneview_redfish_toolkit/blueprints/computer_system_collection.py

```diff
@@ -48,8 +48,15 @@ def get_computer_system_collection():
             raise OneViewRedfishResourceNotFoundError(
                 "server-hardware-list", "Resource")
 
+        # Filter server profiles that has a profile applied
```

Comment:
Filter servers that have a profile applied

Reply:
We could go ahead and change the name of the list as well.